src/RTCRtcpParameters.ts

```diff
@@ -1,14 +1,23 @@
+export interface RTCRtcpParametersInit {
```

Comment:
typo, extra c

Reply:
Nope, that's correct because it's the RTCP parameters, not the RTP ones. Confusing, I know!